lib/transform/standardize.js

```diff
@@ -75,11 +74,11 @@ class Standardizer {
     }
 
     if (keywords.length > 1) {
-      throw new BadRequestError('Invalid filter syntax. Filters must have one keyword only');
+      throw new Error('Invalid filter syntax. Filters must have one keyword only');
     }
 
     if (!this.allowedKeywords.has(keywords[0])) {
-      throw new BadRequestError(`Unknown DSL keyword: ${keywords[0]}`);
+      throw new Error(`Unknown DSL keyword: ${keywords[0]}`);
```

Comment:
Can we have an additional field containing the keyword name ? 
Otherwise we have to extract it from the error message here https://github.com/kuzzleio/kuzzle/blob/master/lib/api/controllers/baseController.js#L89

Reply:
Yes, this is planned, just not in this PR. I try to separate different issues to have a clear v4.0 changelog in the end.